pkgs/by-name/or/oreo-cursors-plus/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  ruby,
+  inkscape,
+  xorg,
+  cursorsConf ? null, # If set to a string, overwrites contents of './cursors.conf'
+}:
+with lib;
```

Comment:
Filewide `with lib;` statement can be somewhat regarded as an antipattern. I do not think `lib` is used enough here to warrant it. No hard requirement though.